### DISCUSSION TITLE
How to run a cron job when server starts on production in next.js?

### DISCUSSION THREAD

**Discussion by arslan990** [vercel/next.js#33268] [Help]
I define a function called Cronjobs, where my cron jobs is running in dev development(npm run dev). I want to run this function when server starts and it will do it is duty. Now the issue I'm facing here it runs on development but when I deploy my app or When I hit command npm run build then npm run start with custom server that function is not called. So please help me out how can I call a function once when server starts.

**Comment by tonyhb** [comment]
Here's a tutorial: https://www.inngest.com/blog/run-nextjs-functions-in-the-background
